approval_polls/models.py

```diff
@@ -44,6 +44,33 @@ def voters(self):
     def __unicode__(self):
         return self.question
 
+    def can_edit(self):
+        return self.total_ballots() == 0
+
+    def add_choices(self, ids, choice_data):
```

Comment:
I am a little hesitant about these methods that encourage passing `request.POST` right to the model. It seems like It might violate some kind of separation of concerns best practices or something. If this doesn't bother anyone else I'll say no more.